### ISSUE TITLE
Lies in Privacy Policy & Consider ditching CloudFlare

### ISSUE THREAD

**Issue by lettersix1** [languagetool-org/languagetool#6556]
Your Add-on Option page

> LanguageTool API server URL:
> Cloud server (languagetool.org) ([Privacy Policy](https://languagetool.org/privacy/))

---

1. **We don't store the text**

By connecting to languagetool.org you are connecting to CloudFlare therefore it [can read anything](http://crimeflare.eu.org).

2. **All of our servers are hosted in Germany.**

CloudFlare edge servers are worldwide and most of them are in the USA.

3. **your IP address is stored in an abbreviated form (like 192.168.xxx.xxx)**

You can log in to https://dash.cloudflare.com and see the real IP address with ID.

4. **This website uses Google Analytics**

Yet, you did not mention "This website uses CloudFlare."

---

Bottom line
1. Please consider ditching CloudFlare.
2. Please review and update your privacy policy.

**Comment by danielnaber** [comment]
> By connecting to languagetool.org you are connecting to CloudFlare therefore it [can read anything](http://crimeflare.eu.org/).

Do your research properly and you'll see that the user's text data is sent to **api**.languagetool.org, which is not connected to Cloudflare. Only the website files themselves are served via Cloudflare.
